src/executor/README.md

```diff
@@ -60,4 +60,10 @@ To run the executor using docker the following steps are required.
 
     This runs the docker container and exposes the port `3500`. The container is now in the `flooq-network`.
 
-4. Visit `http://localhost:3500` to verify that the container is running.
\ No newline at end of file
+4. Visit `http://localhost:3500` to verify that the container is running.
+
+## Environment
+### Variables
+- `API_URL`
+- `API_PORT`
```

Comment:
I agree that it would be more convenient, but this is just how Kubernetes handles internal connections (at least to my current knowledge) and I haven't seen any elegant way to concat them.

I'll keep it in mind, but I can't promise anything.